cylc/uiserver/app.py

```diff
@@ -421,6 +413,21 @@ def __init__(self, *args, **kwargs):
             workflows_mgr=self.workflows_mgr,
         )
 
+    @property
+    def config_file_paths(self) -> List[str]:
+        if self._config_file_paths is None:
+            ret = get_conf_dir_hierarchy(
+                [
+                    SITE_CONF_ROOT,  # site configuration
+                    USER_CONF_ROOT,  # user configuration
+                ], filename=False
+            )
+            # Next include currently needed for directory making
+            ret.insert(0, str(Path(uis_pkg).parent))  # packaged config
+            ret.reverse()
+            self._config_file_paths = ret
+        return self._config_file_paths
```

Comment:
This change does not appear to be necessary, the only bit that's needed is the `autouse`?

Reply:
Couldn't get it to work without this change. Not sure why, but pyright is not happy with overriding a property with a class attribute anyway (see https://github.com/microsoft/pyright/issues/3646#issuecomment-1172558289)

Reply:
Monkeypatching seems to work fine for me (with your config)?

I found the `config_file_paths` definition in Jupyter_Core and it is indeed a property so this makes sense :+1:.